### PR TITLE
eksctl 0.25.0

### PR DESCRIPTION
**Pull request by crunchtime-ali**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by crunchtime-ali** [comment]
I added `head` per request from an `eksctl` contributor here https://github.com/weaveworks/eksctl/pull/2462#discussion_r458736831

File: Formula/eksctl.rb [hunk 1]
```diff
@@ -2,9 +2,10 @@ class Eksctl < Formula
   desc "The official CLI for Amazon EKS"
   homepage "https://eksctl.io"
   url "https://github.com/weaveworks/eksctl.git",
-      tag:      "0.24.0",
-      revision: "f23f8d7d75943d1a62c87ce2f3d455da7af741fa"
+      tag:      "0.25.0",
+      revision: "673de795554c4138baeda442268026c2b22c3bd6"
   license "Apache-2.0"
+  head "https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_Darwin_amd64.tar.gz"
```
Comment (dawidd6) [inline Formula/eksctl.rb:8]:
```suggestion
  head "https://github.com/weaveworks/eksctl.git"
```

Comment (crunchtime-ali) [reply Formula/eksctl.rb:8]: @dawidd6 They used this URL in their own tap to be able to install the latest RC (or stable) version. Is that ok or does it have to be the head revision in the repo as you suggest?

Comment (dawidd6) [reply Formula/eksctl.rb:8]: It has to be as suggested. See other formulae.

Comment (crunchtime-ali) [reply Formula/eksctl.rb]: ok done

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**